cmake/generic.cmake

```diff
@@ -331,3 +331,52 @@ function(go_test TARGET_NAME)
   add_custom_target(${TARGET_NAME} ALL DEPENDS ${TARGET_NAME}_timestamp ${go_test_DEPS})
   add_test(${TARGET_NAME} ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME})
 endfunction(go_test)
+
+function(pb_cc_library TARGET_NAME)
```

Comment:
@reyoung I decided not to call `find_package(Protobuf)`, because when we do cross-compiling, it may find the `protobuf` library of the host. That must be avoided.